content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -266,9 +266,22 @@ This ensures that DaemonSet pods are never evicted due to these problems.
 
 ## Taint Nodes by Condition
 
-The node lifecycle controller automatically creates taints corresponding to
-Node conditions with `NoSchedule` effect.
-Similarly the scheduler does not check Node conditions; instead the scheduler checks taints. This assures that Node conditions don't affect what's scheduled onto the Node. The user can choose to ignore some of the Node's problems (represented as Node conditions) by adding appropriate Pod tolerations.
+The node controller automatically creates taints with a `NoSchedule` effect for
```

Comment:
Let's say: `the control plane, through the node controller`
Is there a link or glossary entry we can add for node controller?

Reply:
Nothing for the node controller specifically but there's a general "controller" one I will add.